### PR TITLE
fix race in serializing sandbox to string

### PR DESCRIPTION
**Pull request by LK4D4**
ping @mrjana @aboch @sanimej

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -868,8 +868,9 @@ func (c *controller) NewSandbox(containerID string, options ...SandboxOption) (s
 		if s.containerID == containerID {
 			// If not a stub, then we already have a complete sandbox.
 			if !s.isStub {
+				err = types.ForbiddenErrorf("container %s is already present: %v", containerID, s)
 				c.Unlock()
-				return nil, types.ForbiddenErrorf("container %s is already present: %v", containerID, s)
```
Comment (aboch) [inline controller.go]: Given this is a very unlikely error and given `sandbox` does not implement the `String()` method, there is no much value in printing the whole raw structure. I am thinking we can avoid the issue and just change the error format to 
`(container %s is already present in sandbox %s", containerID, s.ID())`

**Comment by LK4D4** [comment]
@aboch Updated!

**Comment by aboch** [comment]
LGTM

**Comment by sanimej** [comment]
LGTM

**Pull request merged**